mappings/UofM.py

```diff
@@ -75,6 +70,40 @@ def formatIdentifiers(self):
                     return newISBNList
                 
         return self.record.identifiers
+    
+    def formatSubjects(self):
+        subjectList = []
+
+        if '|' in self.record.subjects:
+            subjectList = self.record.subjects.split('|')
+            newSubjectList = [f'{subject}||' for subject in subjectList] 
+            return newSubjectList
+        else:
+            subjectList.append(f'{self.record.subjects}||')
+            return subjectList
+    
+    def formatRights(self):
+        # Parse rights codes
+        rightsElements = self.record.rights.split('|') if self.record.rights else [''] * 5
+        rightsMetadata = rightsElements[0]
+        licenseMeta = ''
+        statementMeta = ''
+        print(rightsMetadata)
+
+        if rightsMetadata == 'in copyright':
+            licenseMeta = 'in_copyright'
+            statementMeta = 'In Copyright'
```

Comment:
What are the differences between these two fields and what are they used for?

Reply:
The two fields are similar with the statement usually being a longer form version of the license.

Reply:
Both identidy the current rights status of a particular book.